components/ResultsPage/YourAnswers.tsx

```diff
@@ -10,6 +10,7 @@ import { MonthsYears } from '../../utils/api/definitions/types'
 import { Accordion } from '../Forms/Accordion'
 import { fieldDefinitions } from '../../utils/api/definitions/fields'
 import { FieldCategory } from '../../utils/api/definitions/enums'
+import { number } from 'joi'
```

Comment:
not sure about this, needs to be doublecheck

Reply:
imported by accident, removed.